pkg/update/update_cockroach_version_common.go

```diff
@@ -85,19 +86,19 @@ func makeIsCRBPodIsRunningNewVersionFunction(
 
 				// TODO this is not an error but should return a wait status
 				if container.Image != cockroachImage {
-					l.Info("Pod is not updated to current image.")
+					l.V(int(zapcore.DebugLevel)).Info("Pod is not updated to current image.")
```

Comment:
This should be info

Reply:
No

Reply:
See https://github.com/cockroachdb/cockroach-operator/pull/323#pullrequestreview-597673974